proxy-node-gateway/src/main/java/uk/gov/ida/notification/Metrics.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.ida.notification;
+
+import io.prometheus.client.Counter;
+
+public class Metrics {
+
+    public static final String RELEASE_NAME = getReleaseName();
+
+    private static final String RELEASE_NAME_LABEL = "release";
```

Comment:
I feel like this would be better added as a target label in the `ServiceMonitor` resource.  In general it's a Prometheus antipattern for an instance to need to know too much about its own identity; it should just be "a proxy-node instance" and the Prometheus scrape rules (in the ServiceMonitor) should add the context about exactly which proxy-node instance this is, in the form of target labels.

That also means you can get rid of a lot of the wiring around passing the RELEASE_NAME into the app as an envvar.